### PR TITLE
Add "Allow-Control-Allow-Origin" header to all responses from the REST server

### PR DESCRIPTION
**Pull request by tbouron**

### CODE REVIEW

File: rest-server/pom.xml [hunk 1]
```diff
@@ -75,6 +75,11 @@
             <artifactId>javax.servlet-api</artifactId>
             <version>${servlet.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.3.2</version>
```
Comment (m4rkmckenna) [inline rest-server/pom.xml]: Can this be inherited from the parent? ${commons-lang3.version}

Comment (tbouron) [inline rest-server/pom.xml]: It should be a parameter within this `pom.xml`, I'll change it.

However, it doesn't really make sense to put it within the parent pom in my opinion as it's very unlikely that other modules will use this dependency.

Comment (m4rkmckenna) [inline rest-server/pom.xml]: My bad ... I only scanned it thought the parent was 
```
    <parent>
        <groupId>org.apache.brooklyn</groupId>
        <artifactId>brooklyn-downstream-parent</artifactId>
        <version>0.9.0-SNAPSHOT</version>
    </parent>
```

**Comment by tbouron** [comment]
Comment addressed

**Pull request merged**